meta/meta-node/src/cli.rs

```diff
@@ -1,7 +1,6 @@
 use sc_cli::RunCmd;
-use structopt::StructOpt;
```

Comment:
we can remove the dep for `structopt` from toml since we are now using `clap`. and update the code.